package_bag/tests.py

```diff
@@ -71,6 +71,7 @@ def test_discover_bags(self):
         self.assertIsNot(False, discover)
         self.assertEqual(len(discover), expected, "Wrong number of bags processed.")
         self.assertEqual(len(Bag.objects.all()), expected, "Wrong number of bags saved in database.")
+        self.assertEqual(len(listdir(settings.TMP_DIR)), expected, "Wrong number of bags deleted.")
```

Comment:
I realized that I made an error in line 67 above, which should be 
```
expected = len([i for i in listdir(BAG_FIXTURE_DIR) if not i.startswith("invalid_")])
```

I'd also consider changing the message in line 74 so it's something like "Invalid bags were not deleted" since that's really what you're testing here.